src/igvfd/upgrade/modification.py

```diff
@@ -0,0 +1,8 @@
+from snovault import upgrade_step
+
+
+@upgrade_step('modification', '1', '2')
```

Comment:
add notes indicating that the upgrade to include Sp was performed automatically and was not part of original submission